src/System.Data.Common/tests/System/Data/DataViewTest2.cs

```diff
@@ -347,13 +348,14 @@ public void FindRows_ByKeys()
             // FindRows - check data
 
             //check that result is ok
-            bool Succeed = true;
+            bool succeed = true;
             for (int i = 0; i < dvArr.Length; i++)
             {
-                Succeed = (int)dvArr[i]["ChildId"] == (int)drExpected[i]["ChildId"];
-                if (!Succeed) break;
+                succeed = (int)dvArr[i]["ChildId"] == (int)drExpected[i]["ChildId"];
+                if (!succeed)
+                    break;
             }
-            Assert.True(Succeed);
+            Assert.True(succeed);
```

Comment:
This is an odd pattern.  You don't nee to change it in your PR, but it seems like this would have been better just Assert.Equal'ing inside the loop.